server/lib/report_server_web/live/new_report_live/form.ex

```diff
@@ -45,6 +45,8 @@ defmodule ReportServerWeb.NewReportLive.Form do
     |> assign(:filter_type_options, [@filter_type_options])
     |> assign(:filter_options, [[]])
 
+    {:ok, socket}
```

Comment:
You can remove this line - the `handle_params` returns `{:noreply, socket}` on line 50.